### DISCUSSION TITLE
Add Trino connectivity to SQLpage

### DISCUSSION THREAD

**Discussion by jaroet** [sqlpage/SQLPage#624] [Ideas]
OK you think ... just another request for a connector to some database. And yes it is ... but it is Trino :-). 

In my experience (biased) most data intensive companies use a datalake or cloud storage for storing data in .parquet files next to databases. For access and queries a tool like Trino is used to give access to those storages. By supporting Trino you open that  whole big group of companies and people that use cloud buckets, S3 and so on. 

But the fun with Trino is that it connects to a whole list of datasources, see https://trino.io/ecosystem/data-source. So by adding one connector you get many like Bigquery, Redshift, Clickhouse, Snowflake and many more.

**Comment by lovasoa** [comment]
Hello and welcome to SQLPage !

 I'm sorry, I had missed the notification for this discussion, I just saw it. I do agree that the ability to connect to cold storage and other analytics engines would be a great addition to SQLPage. Since there has been a lot of demand for many different databases, what I hope to be able to add is an ODBC backend, which would allow connecting to any SQL database with an ODBC driver (that is almost all of them, including trino).
